compiler/damlc/stable-packages/lib/DA/Daml/StablePackages.hs

```diff
@@ -188,6 +189,29 @@ daInternalAny = package version1_7 $ NM.singleton (emptyModule modName)
           DataRecord [(mkField "getAnyContractKey", TAny), (mkField "getAnyContractKeyTemplateTypeRep", TCon (Qualified PRSelf modName (mkTypeCon ["TemplateTypeRep"])))]
       ]
 
+daInternalInterfaceAnyView :: Package
+daInternalInterfaceAnyView = Package
+  { packageLfVersion = version1_15
+  , packageModules = NM.singleton (emptyModule modName)
+      { moduleDataTypes = datatypes
+      }
```

Comment:
I forgot to mention earlier, we should also add the appropriate selector and worker definitions under `moduleValues`,

```suggestion
daInternalInterfaceAnyView = Package
  { packageLfVersion = version1_15
  , packageModules = NM.singleton (emptyModule modName)
      { moduleDataTypes = datatypes
      { moduleValues = values
      }
```

Reply:
done